src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -137,6 +140,15 @@ const formConfig = {
     dependencies: [services.evss, services.emis, services.mvi, services.vet360],
   },
   formId: VA_FORM_IDS.FORM_21_526EZ,
+  saveInProgress: {
+    messages: {
+      inProgress:
+        'Your disability compensation application (21-526EZ) is in progress.',
+      expired:
+        'Your saved disability compensation application (21-526EZ) has expired. If you want to apply for disability compensation, please start a new application.',
+      saved: 'Your disability compensation application has been saved.',
```

Comment:
Is there any way to allow these to be functions? We have to dynamically change the title of the form between "disability compensation claim" and "Benefits Delivery at Discharge" based on some `formData` values - see the `title` function:

```js
title: ({ formData }) => getPageTitle(formData);
```

Reply:
Not a bad idea. However, since the intent of this PR is only to separate the forms library from the application code, I won't add that new functionality in this here. Feel free to make a [forms library request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=meganhkelley&labels=tools-fe%2C+needs-grooming%2C+triage&template=forms-library-request.md), though. :wink:

Reply:
Ok, thanks for considering it.

Reply:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/14726